test/datatype.test.js

```diff
@@ -68,17 +73,24 @@ describe('datatypes', function() {
   it('should keep types when get read data from db', function(done) {
     var d = new Date, id;
 
-    Model.create({
+    var data = {
       str: 'hello', date: d, num: '3', bool: 1, list: ['test'], arr: [1, 'str'],
-    }, function(err, m) {
+    };
+    if (isCassandraConnector) {
+      data.list = JSON.stringify(data.list);
```

Comment:
This should be handled by the Cassandra connector itself to map Array or Object to stringified json.